builder/builder.go

```diff
@@ -1312,6 +1310,35 @@ func (b *Builder) createMixBundleDir() error {
 	return nil
 }
 
+const dnfConfFormat = `[main]
+cachedir=/var/cache/yum/clear/
+keepcache=0
+debuglevel=2
+logfile=/var/log/yum.log
+exactarch=1
+obsoletes=1
+gpgcheck=0
+plugins=0
+installonly_limit=3
+reposdir=/root/mash
+
+[clear]
+name=Clear
+failovermethod=priority
+baseurl=%s/releases/$releasever/clear/x86_64/os/
```

Comment:
Since you have a `%s` here I see this as a pretty good use case for a template instead of a format string, especially if we want to make the conf more configurable in the future.

Reply:
I didn't go that route 'cause it seemed overkill for a single value, but I have no real objection to it and it does make it easier to configure in the future.

Lemme give it a shot.